### PR TITLE
Update Nav Banner for VSCode's launch on Product Hunt

### PR DESCRIPTION
**Pull request by julieg18**
## Demo 

~https://github.com/iterative/gatsby-theme-iterative/assets/43496356/8bd450dc-6a4d-40d1-98d2-340da35c8da4~

image

### CODE REVIEW

File: packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx [hunk 1]
```diff
@@ -6,9 +6,7 @@ export const AlertContent = () => (
     <span role="img" aria-label="rocket">
       🚀
     </span>{' '}
-    New Release!{' '}
-    <Link href="https://iterative.ai/blog/dvc-3.0-ml-experiments-data-versioning/">
-      The DVC 3.0 Stack: Beyond the Command Line
-    </Link>{' '}
+    Help Support Our DVC for VSCode Extension on{' '}
+    <Link href="https://www.producthunt.com/">Product Hunt</Link>!
```
Comment (julieg18) [inline packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: Any comments on improving the text are welcome!

Comment (julieg18) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: Do we have a link directly to VSCode's product hunt page? I'm currently just linking to the product hunt's page. cc @jendefig, @tapadipti

Comment (tapadipti) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: Thanks @julieg18 

Link: https://www.producthunt.com/posts/dvc-extension-for-vs-code

Text: Let's change to "Support our DVC Extension for VSCode on"

Comment (tapadipti) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: @iterative/studio-frontend Does this appear in Studio also? If not, can we pls add it to Studio separately?

File: packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx [hunk 2]
```diff
@@ -6,9 +6,7 @@ export const AlertContent = () => (
     <span role="img" aria-label="rocket">
       🚀
     </span>{' '}
-    New Release!{' '}
-    <Link href="https://iterative.ai/blog/dvc-3.0-ml-experiments-data-versioning/">
-      The DVC 3.0 Stack: Beyond the Command Line
-    </Link>{' '}
+    Help Support Our DVC for VSCode Extension on{' '}
```
Comment (jendefig) [inline packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:9]:
```suggestion
    Help Support Our DVC Extension for VS Code on{' '}
```

Comment (jendefig) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:9]: Correcting for VS Code branding guidelines.

**Review by jendefig — APPROVED** [review]

File: packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx (hunk 1 above)
Comment (jellebouwman) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: @tapadipti I'm not sure what you mean - do you want us to create a similar banner on the Studio homepage?

Comment (mvshmakov) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: We can do as here https://iterativeai.slack.com/archives/C061NJ44KDX/p1700044523833709?thread_ts=1700036891.594709&cid=C061NJ44KDX

Comment (mvshmakov) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx:10]: https://github.com/iterative/studio/pull/8250

File: packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx (hunk 2 above)
Comment (julieg18) [reply packages/gatsby-theme-iterative/src/components/LayoutHeader/Alert/content.tsx]: Good catch!

**Pull request merged**